rustfmt-core/rustfmt-lib/src/formatting/modules.rs

```diff
@@ -178,7 +220,6 @@ impl<'ast, 'sess, 'c> ModResolver<'ast, 'sess> {
     fn insert_sub_mod(
         &mut self,
         sub_mod_kind: SubModKind<'c, 'ast>,
-        _sub_mod: Cow<'ast, ast::Mod>,
```

Comment:
Oops! I made this change during a previous rustc-ap-* round to get the code compiling again and forgot to remove this after :man_facepalming: